analyzers/src/SonarAnalyzer.CFG/ShimLayer/INamedTypeSymbolExtensions.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
-// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
Can we check if we should change this to MIT or not?

Reply:
As far as I understand we need to update the header with license it since it's part of the file.

I've added back the ReadMe file which I deleted by mistake and I've also included the license file in the shim layer folder.

Reply:
Should we also add the license to the CodeGeneration proj?